src/material/core/option/option.html

```diff
@@ -1,19 +1,18 @@
-<mat-pseudo-checkbox *ngIf="multiple" class="mat-mdc-option-pseudo-checkbox"
-    [state]="selected ? 'checked' : 'unchecked'" [disabled]="disabled"></mat-pseudo-checkbox>
+<mat-pseudo-checkbox *ngIf="multiple" class="mat-mdc-option-pseudo-checkbox" [disabled]="disabled"
+    [state]="selected ? 'checked' : 'unchecked'" [attr.aria-hidden]="true"></mat-pseudo-checkbox>
```

Comment:
```suggestion
<mat-pseudo-checkbox *ngIf="multiple" class="mat-mdc-option-pseudo-checkbox" [disabled]="disabled"
    [state]="selected ? 'checked' : 'unchecked'" aria-hidden="true"></mat-pseudo-checkbox>
```
When an attribute is static, you don't need the binding